src/components/Input/styled/index.ts

```diff
@@ -15,3 +21,24 @@ export const Error = styled.Text`
     margin-top: 4;
     color: ${COLOR_ERROR};
 `;
+
+export const Icon = styled.View<Props>`
+    position: absolute;
+    top: 14px;
```

Comment:
We should not pass a top value for the icon, we have to center the content for the icon with `display: flex` `align-item: center` `justify-content: center`